Projects/Server/Utilities/SplitMix64.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Runtime.CompilerServices;
+

```

Comment:
You can probably just jam this class into the xoroshiro file. That's the only place it's used and it's small.